mne/io/reference.py

```diff
@@ -231,7 +231,8 @@ def add_reference_channels(inst, ref_channels, copy=True):
     return inst
 
 
-def set_eeg_reference(inst, ref_channels=None, copy=True):
+@verbose
+def set_eeg_reference(inst, ref_channels=None, copy=True, verbose=None):
```

Comment:
This is still useful to have as an option. But again need to update the arg list